style.css

```diff
@@ -233,6 +233,119 @@ header .logo {
   transform: scale(1.5);
 }
 
+/* Modals */
+
+.detailSection {
+  position: fixed;
+  display: none;
+  justify-content: center;
+  top: 6%;
+  width: 90vw;
+  margin-right: 2rem;
+  margin-left: 1rem;
+  overflow-y: scroll;
+}
+
+.details {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  align-items: center;
+  width: 100%;
+  margin-left: 2rem;
+  margin-right: 2rem;
+  padding-right: 2rem;
+  padding-left: 1rem;
+  padding-top: 3rem;
+  justify-content: center;
+  padding-bottom: 3rem;
+}
```

Comment:
Great job team 🤝

- [ ] As requested by the previous reviewer, kindly ensure your design doesn't touch the base of your screen. You may consider following the suggestions by the previous reviewer to fix this issue. Kindly apply this to both mobile and desktop screens.
- Your design
![your-base](https://github.com/ulises2607/Portfolio/assets/63917725/c906b9ed-26da-42e1-9df2-100b090e59d8)